supporting_content/dimensions/servicecategory.md

```diff
@@ -0,0 +1,82 @@
+# Column: ServiceCategory
+
+## Example provider mappings
+
+Current column mappings found in available data sets:
+
+| Provider  | Data set                 | Column                                     |
+| --------- | ------------------------ | ------------------------------------------ |
+| AWS       | CUR                      | None                                       |
+| GCP       | Big Query Billing Export | None                                       |
+| Microsoft | Cost details             | ServiceFamily is close, but based on usage |
```

Comment:
not sure if, in case of Azure, serviceName is a good match for providers mapping, especially if use consumedService for serviceName mapping, which I like

Reply:
ServiceFamily is technically the right answer. The difference is just that the values will need to change in some cases based on the resource type.